integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/MessageSender.java

```diff
@@ -65,10 +63,8 @@ public MessageSender(TransportFactory transportFactory, Adresseregister adresser
         this.serviceRegistryLookup = serviceRegistryLookup;
     }
 
-    private Avsender createAvsender(String identifier, ServiceIdentifier serviceIdentifier) throws MessageContextException {
-        Certificate certificate = getCertificate(identifier, serviceIdentifier);
-        PrivateKey privatNoekkel = keyInfo.loadPrivateKey();
-        return Avsender.builder(new Organisasjonsnummer(identifier), new Noekkelpar(privatNoekkel, certificate)).build();
+    private Avsender createAvsender(String identifier) throws MessageContextException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'no.difi.meldingsutveksling.noarkexchange.MessageContextException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)